.github/workflows/scorecard.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        uses: actions/checkout@v4
```

Comment:
[Security scanning](https://github.com/TBD54566975/web5-js/security/code-scanning) checks for pinned dependencies using hash so pin to a v4 using the commit hash of the GitHub action.

> For GitHub workflows used in building and releasing your project, pin dependencies by hash.